pom.xml

```diff
@@ -88,33 +88,53 @@
         <module>server</module>
     </modules>
 
+    <dependencyManagement>
+        <dependencies>
```

Comment:
Why were these additional dependencies added?

Reply:
And why here?
We add these common dependencies (like `log4j`) to the server module so that all the plug-ins can use them.

Reply:
The depedencyManagement section controls versions of (transitive) dependencies used. These few are needed here to upgrade vulnerable dependencies of shrinkwrap (commons-compress, jsoup, log4j-api). If the dependencyManagement section is added to the POM of fasten/core it doesn't control the versions of modules under fasten/analyzer, annoyingly. So it needs to go here, in the parent POM.

Reply:
I see!

Reply:
Oh, I was too fast in skimming the changes and missed this detail...